python/README.rst

```diff
@@ -194,4 +194,6 @@ Due to limitations in the current version of boost-python, some manual changes m
 
 .. code-block:: bat
 
-    > python setup.py --vcpkg-root=[vcpkg-dir] install
\ No newline at end of file
+    > python setup.py --vcpkg-root=[vcpkg-dir] install
+    
+.. _experimental Windows CMake build: https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/README.Windows.md#experimental-cmake-build
```

Comment:
It would be better to link here: https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Dependencies#experimental-cmake-build-system-on-windows